src/components/Medicine/MedicineAdministrationSheet/AdministrationTableRow.tsx

```diff
@@ -59,6 +59,25 @@ export default function MedicineAdministrationTableRow({
       key: `${prescription.last_administration?.administered_date}`,
     },
   );
+  const TableContent = () => (
```

Comment:
proper names for components are appreciated. Also consider keeping this defined outside function and pass prescription as prop.

```suggestion
  const DosageFrequencyInfo = () => (
```